packages/twenty-front/src/modules/object-record/record-table/record-table-cell/hooks/useMoveSoftFocusToCurrentCellOnHover.ts

```diff
@@ -1,29 +1,32 @@
 import { useRecoilCallback } from 'recoil';
 
+import { useSetSoftFocus } from '@/object-record/record-table/record-table-cell/hooks/useSetSoftFocus';
+import { TableCellPosition } from '@/object-record/record-table/types/TableCellPosition';
 import { currentHotkeyScopeState } from '@/ui/utilities/hotkey/states/internal/currentHotkeyScopeState';
 import { getSnapshotValue } from '@/ui/utilities/recoil-scope/utils/getSnapshotValue';
 
-import { TableHotkeyScope } from '../../types/TableHotkeyScope';
-
 import { currentTableCellInEditModePositionComponentState } from '@/object-record/record-table/states/currentTableCellInEditModePositionComponentState';
 import { isTableCellInEditModeComponentFamilyState } from '@/object-record/record-table/states/isTableCellInEditModeComponentFamilyState';
+import { AppHotkeyScope } from '@/ui/utilities/hotkey/types/AppHotkeyScope';
 import { useRecoilComponentCallbackStateV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentCallbackStateV2';
-import { useSetSoftFocusOnCurrentTableCell } from './useSetSoftFocusOnCurrentTableCell';
+import { TableHotkeyScope } from '../../types/TableHotkeyScope';
 
-export const useMoveSoftFocusToCurrentCellOnHover = () => {
-  const setSoftFocusOnCurrentTableCell = useSetSoftFocusOnCurrentTableCell();
+export const useMoveSoftFocusToCurrentCellOnHover = (recordTableId: string) => {
+  const setSoftFocus = useSetSoftFocus(recordTableId);
 
   const currentTableCellInEditModePositionState =
     useRecoilComponentCallbackStateV2(
       currentTableCellInEditModePositionComponentState,
+      recordTableId,
     );
   const isTableCellInEditModeFamilyState = useRecoilComponentCallbackStateV2(
     isTableCellInEditModeComponentFamilyState,
+    recordTableId,
   );
 
-  return useRecoilCallback(
+  const moveSoftFocusToCurrentCell = useRecoilCallback(
```

Comment:
I wouldn't use the naming "current" since we're providing a cellPosition in the parameters.